src/HPLTokenHandler.mo

```diff
@@ -0,0 +1,405 @@
+import RBTree "mo:base/RBTree";
+import Principal "mo:base/Principal";
+import Iter "mo:base/Iter";
+import Option "mo:base/Option";
+import Int "mo:base/Int";
+import AssocList "mo:base/AssocList";
+import Blob "mo:base/Blob";
+import Nat8 "mo:base/Nat8";
+import Array "mo:base/Array";
+import List "mo:base/List";
+import Time = "mo:base/Time";
+import Nat = "mo:base/Nat";
+import Text = "mo:base/Text";
+import R "mo:base/Result";
+
+import CircularBuffer "CircularBuffer";
+import Error "mo:base/Error";
+
+module HPLTokenHandler {
+
+  // https://github.com/research-ag/hpl/#readme
+  public module HPL {
+    public type AssetId = Nat;
+    public type SubaccountId = Nat;
+    public type VirtualAccountId = Nat;
+    public type VirtualAccountUpdateObject = { backingSubaccountId: SubaccountId; assetBalance: Nat };
```

Comment:
Do we really need this? Can we just have two arguments wherever it is needed?

Reply:
Actually we do not need this type here at all. Removed